Example/Core/Tests/FIRAppAssociationRegistrationUnitTests.m

```diff
@@ -52,8 +50,8 @@ - (void)testPassObject {
   id result = [FIRAppAssociationRegistration registeredObjectWithHost:host
                                                                   key:kKey
                                                         creationBlock:^id _Nullable() {
-    return obj;
-  }];
+                                                          return obj;
```

Comment:
FWIW, the rules about block formatting has been removed from Google Objective-C style guide on May 9, 2014. The reason cited was "there are too many flavors, tools provide no help, we’ve never had broad consensus, and consistency is less important to readability than is formatting a block’s code appropriate to the context in which it is used".

I don't mean to block this change -- feel free to proceed. My concern is that there would too much damage if this block formatting is to apply to Auth code one day, where trailing blocks are used extensively.